### ISSUE TITLE
false positive with doseq

### ISSUE THREAD

**Issue by borkdude** [borkdude/clj-kondo#142]
```
$ echo '(doseq [send [inc]] (send 1))' | clj-kondo --lint -
<stdin>:1:21: error: wrong number of args (1) passed to clojure.core/send
```

**Comment by borkdude** [comment]
Fixed with https://github.com/borkdude/clj-kondo/commit/540716712cf9731c6bb694d4df325c30197b5ba5

**Comment by borkdude** [comment]
@marcomorain this should now be fixed in https://github.com/borkdude/clj-kondo/releases/tag/v2019.05.12-alpha
